lua/neo-tree/sources/common/components.lua

```diff
@@ -284,6 +288,9 @@ M.icon = function(config, node, state)
       icon = devicon or icon
       highlight = hl or highlight
     end
+    if node.is_link then
```

Comment:
```suggestion
    if node.is_link and config.file_symlink then
```

What if someone sets this to `nil` because they don't want to show a symlink icon?